libraries/TGRSIAnalysis/TAngularCorrelation/TAngularCorrelation.cxx

```diff
@@ -729,9 +1227,52 @@ Int_t TAngularCorrelation::GenerateMaps(std::vector<Int_t> &arraynumbers, std::v
    fNumIndices = fAngleMap.size();
    fIndexMap = GenerateIndexMap(arraynumbers,distances,fAngleMap);
    fWeights = GenerateWeights(arraynumbers,distances,fIndexMap);
+   fFoldedAngles = GenerateFoldedAngles(fAngleMap);
+   fFoldedAngularIndexes = GenerateFoldedIndexes(fFoldedAngles, fAngleMap);
+   fFoldedAngularWeights = GenerateFoldedWeights(fFoldedAngularIndexes, fWeights);
```

Comment:
So this function doesn't need the group specification as input? Because it doesn't look like it's being called here. If it *does* need the group specification, I suggest we modify this somehow so someone could do an AC with just the original indices and not need to specify the groups. I have no objection to calculating the folded weights here though.